Logger/FileLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Logger;
+
+
+namespace Logger 
+{
+	public class FileLogger : BaseLogger
+	{
+		public string _path { get; }
+		public FileLogger(string path)
+        {
+            _path = path;
+            ClassName = nameof(FileLogger);
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            if(!File.Exists(_path)) { throw new SystemException("Path to file does not exist."); }
+            DateTime date = DateTime.Now;
+            string log = $"{date} {ClassName} {Enum.GetName(typeof(LogLevel), logLevel)}: {message}";
```

Comment:
When logging the `LogLevel`, `Enum.GetName(typeof(LogLevel), logLevel) is not needed. just `logLevel` will produce the correct result.
```suggestion
            string log = $"{date} {ClassName} {logLevel}: {message}";
```